src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/burdenEstimateRepository/BurdenEstimateRepositoryTests.kt

```diff
@@ -231,68 +231,6 @@ abstract class BurdenEstimateRepositoryTests : RepositoryTests<BurdenEstimateRep
             modelRunParameterSet = null
     )
 
-
```

Comment:
These unused methods must have been missed when deleting all stochastic related code. Now the table is gone they were failing to compile.